src/components/Layout/Layout.styled.js

```diff
@@ -1,9 +1,7 @@
 import styled from 'styled-components';
 
-const Container = styled.div`
-  margin-top: 100px;
-`;
-const Main = styled.main``;
+const Container = styled.div.attrs((props) => ({ className: `bg-${props.theme}` }))``;
```

Comment:
Good use of dynamic props from `styled-components`!

However, I think that maybe using both styled-components and bootstrap can be a little redundant. Maybe if you're going to use boostrap you could just have a function that computes the classnames.